test/drivers/CMakeLists.txt

```diff
@@ -60,6 +60,8 @@ if(${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64" AND ENABLE_IA32_TESTS)
     )
 endif()
 
+add_compile_options(${FALCOSECURITY_LIBS_USERSPACE_COMPILE_FLAGS})
```

Comment:
Can we use `target_compile_options` and `target_link_options`?  Or, if you are willing to add the compile/link options to all targets, we can just do it once in the CompilerFlags.cmake.

Reply:
Basically what this wants to do is to set some flags only for userspace components. The drivers must not be compiled with sanitizer flags or they will crash. Using `target_compile_options` means that the options will need to be specified for every single target, meaning that if tomorrow we add another target under those directories it will not use those flags unless the author remembers to add `target_compile_options`

Reply:
I see, makes sense then! Thanks!